R/addins.R

```diff
@@ -1,25 +1,50 @@
-#' Style the active file
+
+#' Stylers for RStudio Addins
+#'
+#' Helper functions for styling via RStudio Addins.
+#'
+#' @section Auto-Save Option:
+#' By default, both of the RStudio Addins will apply styling to the (selected)
+#' file contents without saving changes. Automatic saving can be enabled by
+#' setting the environment variable \code{save_after_styling} to \code{TRUE}.
 #'
-#' Helper function for RStudio Addin.
+#' Consider setting this in your \code{.Rprofile} file if you want to persist
+#' this setting across multiple sessions. Untitled files will always need to be
+#' saved manually after styling.
+#'
+#' @name styler_addins
+#' @family stylers
+#' @seealso [Sys.setenv()]
+NULL
+
+#' @describeIn styler_addins Styles the active file
 style_active_file <- function() {
   context <- get_rstudio_context()
-  style_file(context$path, style = tidyverse_style)
+  out <- style_text(context$contents)
+  rstudioapi::modifyRange(
+    c(1, 1, length(out) + 1, 1),
+    paste0(out, collapse = "\n"), id = context$id
+  )
+  if (Sys.getenv("save_after_styling") == TRUE && context$path != "") {
+    rstudioapi::documentSave(context$id)
+  }
 }
 
 
-#' Style the highlighted region
-#'
-#' Helper function for RStudio Addin. This function is complicated because of
-#' one thing: You can highlight also just parts of lines.
+
+#' @describeIn styler_addins Styles the highlighted region
 #' @importFrom rlang seq2
```

Comment:
I think the `rlang` import is redundant. Can you remove as part of this PR?

Reply:
Sure.